package.json

```diff
@@ -199,6 +200,7 @@
     "bundle:css": "node-sass content-src/main.scss -o data/content",
     "bundle:html": "node ./bin/generate-html.js > data/content/activity-streams.html",
     "postbundle": "du -hs ./data/content/*",
+    "bundlestats": "NODE_ENV=production webpack --json | webpack-bundle-size-analyzer",
```

Comment:
Aha! I was wondering how you were generating those charts... I've only ever used `webpack-bundle-size-analyzer`, I didn't notice the name difference